src/class-elements/syntax/valid/cls-decl-elements-valid-syntax-block.template

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-ClassElement
+name: class declaration inside block
+path: language/statements/class/elements/syntax/valid/inside-function-
+features: [class]
+---*/
+
+{
+  class C /*{ heritage }*/{
+    /*{ elements }*/
+  }
+}
```

Comment:
a class declaration inside a block is a class declaration and we should limit the tests to itself alone, no expanding to class elements.

I recommend removing this like in src/class-elements/syntax/valid/cls-decl-elements-valid-syntax-inside-function.template